content/03.ml-concepts.md

```diff
@@ -1,2 +1,14 @@
 ## Rule 1: Concepts that apply to machine learning also apply to deep learning
+Deep learning is a distinct subfield of machine learning, but it is still a subfield.
+Deep learning has proven to be an extremely powerful paradigm capable of outperforming “traditional” machine learning approaches, but it is not immune to the many limitations inherent to machine learning.
+Many best practices for machine learning apply to deep learning as well.
+For instance, deep supervised learning models should be trained, tuned, and tested on non-overlapping datasets.
+Those developing deep learning models should first investigate their data.
+When coupled with imprudence, data that is biased, skewed, or of low quality will produce models of dubious performance and limited generalizability.
+Biases in testing data can also unduly influence measures of model performance.
+For example, many conventional metrics for classification (e.g. area under the receiver operating characteristic curve or AUROC) have limited utility in cases of extreme class imbalance.
```

Comment:
Maybe a good paper to cite here: 

Korotcov A, Tkachenko V, Russo DP, Ekins S: Comparison of Deep Learning With Multiple Machine Learning Methods and Metrics Using Diverse Drug Discovery Data Sets. Molecular Pharmaceutics 2017, 14:4462–4475.